github/gists_test.go

```diff
@@ -281,6 +281,48 @@ func TestGistsService_Edit_invalidID(t *testing.T) {
 	testURLParseError(t, err)
 }
 
+func TestGistsService_ListCommits(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/gists/1/commits", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		testFormValues(t, r, nil)
+		fmt.Fprint(w, `
+			[
+				{"version": "1",
+				 "user": {"id": 1},
+				 "change_status": {
+				 	"deletions": 0,
+				 	"additions": 180,
+				 	"total": 180
+				 },
+				 "commited_at": "2010-01-01T00:00:00Z"
+				}
```

Comment:
The formatting of this JSON looks kinda off. Can you make it look more consistent with what's in this file, or https://github.com/google/go-github/blob/45cbf2313cdd7e84ba9ef011885adbd72d6e6604/github/git_refs_test.go#L65-L84?